encrypted-media-respec.html

```diff
@@ -2301,6 +2308,29 @@
             <li><p>Resolve <var>promise</var>.</p></li>
           </ol>
         </section>
+        
+        <section id="media-key-session-destroyed">
+          <h4>MediaKeySession destroyed</h4>
+          <p>
+            This algorithm performs steps that are necessary when a <a>MediaKeySession</a> is destroyed without
```

Comment:
"without" somewhat implies something wrong happened. I don't think this is unexpected. I propose we simplify this a little:

>`... when a <a>MediaKeySession</a> is destroyed **and** the <a def-id="session-closed-algorithm"></a> algorithm **has not** been run.`